### PR TITLE
stackrox: more 3.72/3.73 release branch coverage

### PR DESCRIPTION
**Pull request by gavin-stackrox**
Adds a few missing IMO useful tests.
aa71653b2fe 3.72: add version-compatibility
9d45067d242 3.73: add kernel, race-condition, version-compatibility

### CODE REVIEW

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.73.yaml [hunk 1]
```diff
@@ -288,6 +292,16 @@ tests:
     test:
     - ref: stackrox-e2e
   timeout: 5h0m0s
+- as: gke-race-condition-qa-e2e-tests
+  steps:
+    test:
+    - ref: stackrox-e2e
+  timeout: 5h0m0s
+- as: gke-version-compatibility-tests
```
Comment (msugakov) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.73.yaml:300]: They were partial at 3.73 cut-off time so we could try... Although I did not enable in my https://github.com/openshift/release/pull/36303

**Review by msugakov — COMMENTED** [review]
I found your PR after I created mine https://github.com/openshift/release/pull/36303
I think, mine may be more complete (except of version compatibility tests). Which way we go?

**Comment by gavin-stackrox** [comment]
> I found your PR after I created mine #36303 I think, mine may be more complete (except of version compatibility tests). Which way we go?

Lets go with #36303.

**Pull request closed without merging**